Sources/SwiftTestCodecovLib/Aggregate.swift

```diff
@@ -33,8 +33,26 @@ public struct Aggregate: Encodable {
     public init(
         coverage: CodeCov,
         property: CodeCov.AggregateProperty,
-        includeDependencies: Bool
+        includeDependencies: Bool,
+        stripTestFiles: Bool
     ) {
+        var coverage = coverage
+
+        if stripTestFiles {
+            var nonTestDataSet = [CodeCov.Data]()
+            for datum in coverage.data {
+                let nonTestFiles = datum.files.filter({ file in
+                    !(file.filename.lowercased().contains("tests") || file.filename.lowercased().contains("test"))
```

Comment:
```suggestion
                    !(file.filename.lowercased().contains("test"))
```
Since the word "tests" contains the word "test"

Reply:
Aah yes. Makes sense. Changing this